cardinal/server/server_test.go

```diff
@@ -203,6 +210,95 @@ func TestHandleTransactionWithNoSignatureVerification(t *testing.T) {
 	assert.Equal(t, 1, count)
 }
 
+func TestHandleSwaggerServer(t *testing.T) {
+	count := 0
+	endpoint := "move"
+	w := inmem.NewECSWorldForTest(t)
+	sendTx := ecs.NewTransactionType[SendEnergyTx, SendEnergyTxResult](endpoint)
+	assert.NilError(t, w.RegisterTransactions(sendTx))
+	w.AddSystem(func(world *ecs.World, queue *ecs.TransactionQueue, _ *ecs.Logger) error {
+		txs := sendTx.In(queue)
+		assert.Equal(t, 1, len(txs))
+		tx := txs[0]
+		assert.Equal(t, tx.Value.From, "me")
+		assert.Equal(t, tx.Value.To, "you")
+		assert.Equal(t, tx.Value.Amount, uint64(420))
+		count++
+		return nil
+	})
+
+	//TODO: next
+	// Queue up a CreatePersonaTx
+	//personaTag := "foobar"
+	//signerAddress := "xyzzy"
+	//ecs.CreatePersonaTx.AddToQueue(w, ecs.CreatePersonaTransaction{
+	//	PersonaTag:    personaTag,
+	//	SignerAddress: signerAddress,
+	//})
+	//readPersonaRequest := ReadPersonaSignerRequest{
+	//	PersonaTag: personaTag,
+	//	Tick:       0,
+	//}
+	//create readers
+	type FooRequest struct {
+		ID string
+	}
+	type FooReply struct {
+		Name string
+		Age  uint64
+	}
+
+	expectedReply := FooReply{
+		Name: "Chad",
+		Age:  22,
+	}
+	fooRead := ecs.NewReadType[FooRequest, FooReply]("foo", func(world *ecs.World, req FooRequest) (FooReply, error) {
+		return expectedReply, nil
+	}, ecs.WithReadEVMSupport[FooRequest, FooReply])
```

Comment:
Is ecs.WithReadEVMSupport required for this test?

Reply:
It probably will be for the other routes I will be implementing.